enterprise/frontend/src/metabase-enterprise/tools/containers/ErrorDetail.jsx

```diff
@@ -0,0 +1,185 @@
+import React from "react";
+import { t } from "ttag";
+import PropTypes from "prop-types";
+import { getIn } from "icepick";
+
+import { formatColumn, formatValue } from "metabase/lib/formatting";
+import { CardApi } from "metabase/services";
+import Button from "metabase/components/Button";
+import Link from "metabase/components/Link";
+import Question from "metabase-lib/lib/Question";
+import { QuestionResultLoader } from "metabase/containers/QuestionResultLoader";
+import { columnNameToUrl } from "../../audit_app/lib/mode";
+
+const CARD_ID_ROW_IDX = 0;
+const ErrorDrill = ({ clicked }) => {
+  if (!clicked) {
+    return [];
+  }
+
+  const cardId = clicked.origin.row[CARD_ID_ROW_IDX];
+
+  return [
+    {
+      name: "detail",
+      title: `View this`,
+      default: true,
+      url() {
+        return `/admin/tools/errors/${cardId}`;
+      },
+    },
+  ];
+};
+
+export const ErrorMode = {
+  name: "error",
+  drills: () => [ErrorDrill],
+};
+
+function idxToUrl(resRow, resCols, nameToResCol, colName) {
+  const idVal = resRow[nameToResCol[colName]];
+  const urlVal = colName && idVal ? columnNameToUrl[colName](idVal) : "";
+  const linkClass = urlVal === "" ? "" : "text-brand";
+  return [urlVal, linkClass];
+}
+
+function ErrorDetailDisplay(props) {
+  const { result } = props;
+  const resRow = getIn(result, ["data", "rows", 0]);
+  const resCols = getIn(result, ["data", "cols"]);
+  if (resRow && resCols) {
+    const nameToResCol = resCols.reduce(
+      (obj, x, idx) => Object.assign(obj, { [x.name]: idx }),
+      {},
+    );
+
+    const linkColumns = [
```

Comment:
I'm wondering if it is possible to use a similar way to the `AuditTable` defining these rows in visual settings like here `enterprise/frontend/src/metabase-enterprise/audit_app/lib/cards/queries.js` and then use a drill to make them clickable

Reply:
I don't understand drills well enough to make them all heterogeneous so that the title is also clickable, is the problem.